net.sourceforge.paulstretch.yaml

```diff
@@ -0,0 +1,129 @@
+  - --filesystem=/media
+  # temp files
+  - --filesystem=/tmp
+  # Fltk does not support wayland (yet)
+  #- --socket=wayland
+  #- --socket=fallback-x11
+  # X11
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  # Playback
+  - --socket=pulseaudio
+  # Jack
+  - --filesystem=xdg-run/pipewire-0
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+modules:
+  - name: audiofile
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://audiofile.68k.org/audiofile-0.3.6.tar.gz
+        sha256: cdc60df19ab08bfe55344395739bb08f50fc15c92da3962fac334d3bff116965
+      - type: patch
+        paths:
+          - patches/audiofile/01_gcc6.patch
+          - patches/audiofile/02_hurd.patch
+          - patches/audiofile/03_CVE-2015-7747.patch
+          - patches/audiofile/04_clamp-index-values-to-fix-index-overflow-in-IMA.cpp.patch
+          - patches/audiofile/05_Always-check-the-number-of-coefficients.patch
+          - patches/audiofile/06_Check-for-multiplication-overflow-in-MSADPCM-decodeSam.patch
+          - patches/audiofile/07_Check-for-multiplication-overflow-in-sfconvert.patch
+          - patches/audiofile/08_Fix-signature-of-multiplyCheckOverflow.-It-returns-a-b.patch
+          - patches/audiofile/09_Actually-fail-when-error-occurs-in-parseFormat.patch
+          - patches/audiofile/10_Check-for-division-by-zero-in-BlockCodec-runPull.patch
+          - patches/audiofile/11_CVE-2018-13440.patch
+          - patches/audiofile/12_CVE-2018-17095.patch
+  - name: mxml
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/michaelrsweet/mxml/releases/download/v3.3.1/mxml-3.3.1.tar.gz
+        sha256: 0c663ed1fe393b5619f80101798202eea43534abd7c8aff389022fd8c1dacc32
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share/doc
+      - /share/man
+      - "*.so"
+      - "*.a"
+  - name: portaudio
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://github.com/PortAudio/portaudio/archive/refs/tags/v19.7.0.tar.gz
+        sha256: 5af29ba58bbdbb7bbcefaaecc77ec8fc413f0db6f4c4e286c40c3e1b83174fa0
+  - shared-modules/linux-audio/fftw3f.json
+  - shared-modules/linux-audio/jack2.json
+  - shared-modules/libmad/libmad.json
+  - name: libvorbis
+    buildsystem: autotools
+    cleanup:
+      - "/share/doc/libvorbis*"
+    sources:
+      - type: archive
+        url: https://ftp.osuosl.org/pub/xiph/releases/vorbis/libvorbis-1.3.7.tar.xz
+        sha256: b33cc4934322bcbf6efcbacf49e3ca01aadbea4114ec9589d1b1e9d20f72954b
+      - type: patch
+        path: patches/libvorbis-1.3.7-aotuv-b6.03.patch
+      - type: patch
+        path: patches/libvorbis-1.3.7-aotuv-b6.03-lancer.patch
+  - shared-modules/glu/glu-9.json
+  - name: fltk
+    buildsystem: cmake-ninja
+    build-options:
+      cflags: -O0
+      cxxflags: -O0 # app crashes without this
```

Comment:
no. You should figure why it crashes since it is a non-release of fltk.

Reply:
I fixed all other suggestions but this, because app crashes otherwise.

Ok so I'll leave this PR open till there's a fltk release then.

Reply:
what about the current release of fltk? it works. Given that this is not a recent app.

Reply:
It works but I (and many people) have a HiDPI monitor and it's unusable because of tiny size, and I don't want people to experience that.

Reply:
what do you think other apps using fltk do?

Reply:
They are using fltk 1.3.x and looking awful on HiDPI. Current fltk git is what will become 1.4 and already has support for HiDPI (and in future Wayland support too).

Reply:
if it crashes without -O0 then the code isn't ready for users.